### ISSUE TITLE
Failed to execute CREATE PROCEDURE

### ISSUE THREAD

**Issue by chewlm86** [Bajdzis/vscode-database#55]
Hi,

I get the following error message when i try to execute this SQL to create a Store Procedure on MariaDB. (I have commented the delimiter before executing it)

Error:
`ER_PARSE_ERROR: You have an error in your SQL syntax; `

SQL:
```
-- DELIMITER //
 
DROP PROCEDURE IF EXISTS GetAllPhoneBook;
CREATE PROCEDURE GetAllPhoneBook()
BEGIN
 DECLARE EXIT HANDLER FOR SQLEXCEPTION
 BEGIN
  ROLLBACK;
  SELECT 'An error has occurred, operation rollbacked and the stored procedure was terminated';
 END;
 
 START TRANSACTION;
 SELECT * FROM phone_book;
 COMMIT;
END;
-- //

-- DELIMITER ;
```

**Comment by ul8ksgdmy** [comment]
I've got stuck by exactly the same problem while I created a stored procedure. Is there anyone who can help me?

**Comment by Bajdzis** [comment]
This split method probably make this error.

https://github.com/Bajdzis/vscode-database/blob/master/extension/engine/AbstractServer.js#L50

The best solution will be to add a new button to execute queries that should be interpreted as one.

**Comment by Bajdzis** [comment]
Fixed in 2.2.0 version
